platform/engineering/request-for-comment/2022-02-23-remove-gender-fax-from-mobile-api

```diff
@@ -0,0 +1,23 @@
+RFC: Removing gender and fax number from mobile api
+Date: 2022-02-23
+Related Issue: department-of-veterans-affairs/va-mobile-app#2565
+Background
+Currently gender and fax are both returned from the mobile API and displayed in the app. 
+
+Motivation
+Profile is removing gender and fax number. More details: https://dsva.slack.com/archives/C018V2JCWRJ/p1643395405843779
```

Comment:
I'd like to see some discussion here on the impacts of removing these from Profile. Something along the lines of: The mobile API – a module inside vets-api – must support multiple versions of the app in the wild. We do this through API versioning. If gender and fax are removed from Profile without proper mitigation measures (making this data available from somewhere), the mobile API cannot properly support older versions of the app. The ultimate impact is that app users will see error messages when trying to view or update their personal information.

Reply:
Yeah I can updated that to be more verbose about the situation.